.tekton/main-pull-request.yaml

```diff
@@ -299,6 +299,22 @@ spec:
           image: registry.access.redhat.com/ubi8/ubi:latest
           script: exec "$(workspaces.source.path)/source/scripts/konflux/subscription-manager/subscription-manager-bro.sh" smuggle
 
+    - name: fetch-external-networks
+      runAfter:
+      - clone-repository
+      taskSpec:
+        steps:
+          - name: fetch-external-networks
+            image: registry.access.redhat.com/ubi8-minimal:latest
+            script: |
+              #!/usr/bin/env bash
+              microdnf install -y openssl zip
```

Comment:
Why did you add `openssl`, is it because it brings something needed for `curl` to function?

Reply:
I added it because it was present in the upstream Dockerfile: https://github.com/stackrox/stackrox/blob/master/image/rhel/Dockerfile#L19. 

However, it is not required and I removed it in https://github.com/stackrox/stackrox/pull/10672/commits/fdb6d014a9bd20467448de2dd0565edf0ecad37d